src/goto-diff/goto_diff_parse_options.h

```diff
@@ -27,6 +30,8 @@ class optionst;
 #define GOTO_DIFF_OPTIONS \
   "(json-ui)" \
   OPT_SHOW_GOTO_FUNCTIONS \
+  OPT_GOTO_CHECK \
+  "(cover):" \
```

Comment:
Why is it not necessary to add any of the Java stuff that got added to the help output in here?